script/rules/OriginRules.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity ^0.8.24;
+
+import {IVault, IValidator} from "src/interface/IVault.sol";
```

Comment:
there's no origin on bsc, let's remove it

Reply:
removed